doc/python/mapbox-county-choropleth.md

```diff
@@ -249,5 +252,6 @@ fig.show()
 
 #### Reference
 
-See [function reference for `px.(choropleth_mapbox)`](https://plotly.com/python-api-reference/generated/plotly.express.choropleth_mapbox) or https://plotly.com/python/reference/choroplethmapbox/ for more information about mapbox and their attribute options.
+See [function reference for `px.choropleth_map`](https://plotly.com/python-api-reference/generated/plotly.express.choropleth_map) or https://plotly.com/python/reference/choroplethmap/ for more information about the attributes available.
```

Comment:
See _the_ function reference (?)

Reply:
Yeah, I think that would be more correct, but it's already written this way across all other pages, so I won't update it in this PR.